crates/rpc-types/Cargo.toml

```diff
@@ -19,7 +19,7 @@ alloy-serde.workspace = true
 alloy-genesis.workspace=true
 
 alloy-consensus = { workspace = true, features = ["serde"] }
-alloy-eips = {workspace = true, features = ["std", "serde"]}
+alloy-eips = {workspace = true, features = ["std", "serde", "arbitrary"]}
```

Comment:
arbitrary is optional, should go in the features table

Reply:
Fixed